tutorial/tutorial-gabc-01.html

```diff
@@ -248,7 +248,7 @@
 </pre>
 </div>
 
-<p>Here's the result after processing by gregorio and LaTeX:</p>
+<p>Here's the result after processing by gregorio and <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span>:</p>
```

Comment:
In the section above, `but is put the notes for "-son" before the period` is grammatically incorrect, at least in English.  It should probably be `but the notes for "-son" were put before the period`.